trogdor/src/main/java/org/apache/kafka/trogdor/workload/SustainedConnectionWorker.java

```diff
@@ -60,7 +60,7 @@
 
 public class SustainedConnectionWorker implements TaskWorker {
     private static final Logger log = LoggerFactory.getLogger(SustainedConnectionWorker.class);
-    private static final SystemTime SYSTEM_TIME = new SystemTime();
+    private static final Time SYSTEM_TIME = Time.SYSTEM;
```

Comment:
I think you can inline this variable.

Reply:
You are right,  
After updating `SystemTime` to singleton mode, many class attributes `time` can be modified inline, but I am not sure whether to modify it because it is a large-scale change to modify many files.

Reply:
I don't mean the local variables and instance variables `time`, those can stay as-is, as they can be useful later if someone wants to replace the Time instance for the whole test. 

I meant this duplicate static final variable. Either it should be an instance variable like the rest, or inlined.

Reply:
Thank you

Reply:
Thank you